src/zzz/zh/quickstart.md

```
@@ -119,30 +122,27 @@ D:\ZZZ-OD
-   - 只下载精简安装器、其余内容联网获取：下载 `ZenlessZoneZero-OneDragon-<版本>-Installer.exe`
+   - 下载 `ZenlessZoneZero-OneDragon-<版本>-Full-Environment.zip`，其中包含安装器、启动器、打包时的程序代码、识别模型和离线环境资源
+      - 从 [GitHub Release](https://github.com/OneDragon-Anything/ZenlessZoneZero-OneDragon/releases) 下载，网络要求特殊
+      - 从 [CNB Releases](https://cnb.cool/OneDragon-Anything/ZenlessZoneZero-OneDragon/-/releases) 下载
+      - 从 [Mirror酱](https://mirrorchyan.com/zh/projects?rid=ZZZ-OneDragon&arch=arm64&channel=stable)（需要 Mirror CDK，<font color="red">选择 arm64</font>，对应 `Full-Environment` 包）
 2. 运行安装器
    - 以管理员身份运行 `OneDragon-Installer.exe`，选择符合[安装路径要求](#安装路径要求)的最终安装目录
-   - 不能选择磁盘根目录
-   - 路径必须为纯英文且不能包含空格
    - 目标目录不为空时，安装器会提示内容可能被覆盖；首次安装建议使用空目录
-   - 安装器与目标目录不同时，会按照安装清单搬运并校验文件
 3. 配置源（觉得麻烦？请使用[集成启动器](#集成启动器-推荐-解压即用)）
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)